templates/zerver/help/browse-and-join-streams.md

```diff
@@ -52,5 +52,3 @@ subscribe to (highlighted in red).
     A green checkmark will appear next to the stream that you've just
 subscribed to, and you will now be able to read and send messages in
 that stream.
-
-    ![Subscribe after](/static/images/help/subscribe-after.png)
```

Comment:
Just to confirm, have you deleted `/static/images/help/stream-overview.png` or is it needed at some other place?